src/themes/dashboard/SampleReactTable.js

```diff
@@ -0,0 +1,110 @@
+import React, { useMemo } from 'react';
+import { Heading, Text, Table, Thead, Tbody, Tr, Th, Td, chakra } from '@chakra-ui/react';
+import { TriangleDownIcon, TriangleUpIcon } from '@chakra-ui/icons';
+import { useTable, useSortBy } from 'react-table';
+
+const SampleReactTable = () => {
+  const data = React.useMemo(
+    () => [
+      {
+        name: '1st Name',
+        email: 'example1@email.com',
+        action: 'Remove',
+      },
+      {
+        name: '2nd Name',
+        email: 'example2@email.com',
+        action: 'Remove',
+      },
+      {
+        name: '3rd Name',
+        email: 'example3@email.com',
+        action: 'Remove',
+      },
+      {
+        name: '4th Name',
+        email: 'example4@email.com',
+        action: 'Remove',
+      },
+      {
+        name: '5th Name',
+        email: 'example5@email.com',
+        action: 'Remove',
+      },
+      {
+        name: '6th Name',
+        email: 'example6@email.com',
+        action: 'Remove',
+      },
+    ],
+    [],
+  );
+
+  const columns = useMemo(
+    () => [
+      {
+        Header: 'NAME',
+        accessor: 'name',
+        Cell: props => <Heading size="h4">{props.value}</Heading>,
+      },
+      {
+        Header: 'EMAIL',
+        accessor: 'email',
+      },
+      {
+        id: 'action',
+        accessor: 'action',
+        Cell: props => (
+          <Text fontWeight="semibold" color="#C70E0E">
+            {props.value}
+          </Text>
+        ),
+      },
+    ],
+    [],
+  );
+
+  const renderSortIcon = column => {
+    const { isSorted, isSortedDesc } = column;
+    return isSorted ? (
+      isSortedDesc ? (
+        <TriangleDownIcon aria-label="sorted descending" />
+      ) : (
+        <TriangleUpIcon aria-label="sorted ascending" />
+      )
+    ) : null;
+  };
+
+  const { getTableProps, getTableBodyProps, headerGroups, rows, prepareRow } = useTable({ columns, data }, useSortBy);
+
+  return (
+    <Table {...getTableProps()}>
+      <Thead>
+        {headerGroups.map(headerGroup => (
+          <Tr {...headerGroup.getHeaderGroupProps()}>
+            {headerGroup.headers.map(column => (
+              <Th {...column.getHeaderProps(column.getSortByToggleProps())}>
```

Comment:
imo a nice UI for these interactive headings would be to give them the darker gray bg colour on hover (same as the rows on hover)

I also noticed that all columns are sortable (including the 'remove' column). Would be worth investigating what the standard way of making a column non-sortable is using react-table and implementing that here (along with matching UI, so no hover effect and no cursor:pointer)